modules/iam-user/variables.tf

```
@@ -39,6 +39,12 @@ variable "pgp_key" {
   default     = ""
 }
 
+variable "iam_access_key_status" {
+  description = "Access key status to apply."
+  type        = string
+  default     = "Active"
```

Since this is optional, it should be `null` which the API or the provider will set the actual default value

```suggestion
  default     = null
```